### PR TITLE
add ci

### PR DESCRIPTION
**Pull request by marsishandsome**
close https://github.com/tidb-incubator/TiBigData/issues/39

support running ci by using following command
- `/build`
- `/run-all-tests`

image

image

### CODE REVIEW

**Comment by marsishandsome** [comment]
@sunxiaoguang @humengyu2012 PTAL

File: tidb/src/test/java/org/tikv/tibigdata/tidb/TestSqlUtils.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package org.tikv.bigdata.tidb;
+package org.tikv.tibigdata.tidb;
```
Comment (sunxiaoguang) [inline tidb/src/test/java/org/tikv/tibigdata/tidb/TestSqlUtils.java:1]: Looks like there are tests not using org.tikv.bigdata as package name

**Review by sunxiaoguang — CHANGES_REQUESTED** [review]
Looks like there are certain code not using package name org.tikv.bigdata

File: tidb/src/test/java/org/tikv/tibigdata/tidb/TestSqlUtils.java (hunk 1 above)
Comment (marsishandsome) [reply tidb/src/test/java/org/tikv/tibigdata/tidb/TestSqlUtils.java]: fixed

**Comment by sunxiaoguang** [comment]
> /run-all-tests

Please let us know when CI scripts are ready to be reviewed. Thanks

**Comment by marsishandsome** [comment]
> > /run-all-tests
> 
> Please let us know when CI scripts are ready to be reviewed. Thanks

@sunxiaoguang PTAL

**Review by sunxiaoguang — APPROVED** [review]
LGTM

**Pull request merged**